### ISSUE TITLE
README.md per package.

### ISSUE THREAD

**Issue by csantanapr** [watson-developer-cloud/openwhisk-sdk#8]
This repo holds multiple package.
There should be one README.md per each package.
At high level the README should have:
- table with listing actions, with columns with short description, and parameter names.
- section with parameter that are common across all functions in the package (parameter name, type (JSON Object, boolean, string, number, etc) and description this can be in table format.
- each function should have a section with info about the action, description, parameters and description for parameter.

The repo should refactor with a different directory structure per package, each package directory with a README.md
```
package-name1\
  action1.js
  action1.json
  README.md
package-name2\
  action2.js
  action2.json
  README.md
```

**Comment by bvennam** [comment]
Created a PR with a sample Readme.
https://github.com/watson-developer-cloud/openwhisk-sdk/pull/13

One thing Anwesha & I discussed was the possibility of linking out to the API Reference so that this information is not repeated in two places, and so the readme is not so long.  I linked there from the Parameters for package & actions table, and I think that should be fine.

One thing I don't think is reflected in the api reference is parameters that are common across all functions in the package.  

@csantanapr feel free to weigh in.
